niworkflows/viz/utils.py

```diff
@@ -65,19 +66,24 @@ def save_html(template, report_file_name, unique_string, **kwargs):
 
 
 def as_svg(image, filename='temp.svg'):
-    ''' takes an image as created by nilearn.plotting and returns a blob svg.
-    A bit hacky. '''
+    ''' Takes an image as created by nilearn.plotting and returns the svg with
+        extra headers removed and the width and height style values set to
+        nothing. '''
     image.savefig(filename)
     with open(filename, 'r' if PY3 else 'rb') as file_obj:
         image_svg = file_obj.readlines()
 
-    svg_start = 0
+    svg_start = None
+    svg_lines_corrected = []
     for i, line in enumerate(image_svg):
```

Comment:
Particularly, I find it more readable if we did this this way: https://github.com/oesteban/mriqc/blob/master/mriqc/viz/svg.py#L63